src/algebra/ordered_field.lean

```diff
@@ -181,12 +181,18 @@ namespace nat
 
 variables {α : Type*} [linear_ordered_field α]
 
-lemma inv_pos_of_nat [linear_ordered_field α] {n : ℕ}  : 0 < ((n : α) + 1)⁻¹ :=
+lemma inv_pos_of_nat {n : ℕ}  : 0 < ((n : α) + 1)⁻¹ :=
 inv_pos $ add_pos_of_nonneg_of_pos n.cast_nonneg zero_lt_one
 
-lemma one_div_pos_of_nat [linear_ordered_field α] {n : ℕ} : 0 < 1 / ((n : α) + 1) :=
+lemma one_div_pos_of_nat {n : ℕ} : 0 < 1 / ((n : α) + 1) :=
 by { rw one_div_eq_inv, exact inv_pos_of_nat }
 
+lemma one_div_le_one_div {n m : ℕ} (h : n ≤ m) : 1 / ((m : α) + 1) ≤ 1 / ((n : α) + 1) :=
+by { refine one_div_le_one_div_of_le _ _, exact nat.cast_add_one_pos _, simpa }
+
+lemma one_div_lt_one_div {n m : ℕ} (h : n < m) : 1 / ((m : α) + 1) < 1 / ((n : α) + 1) :=
+by { refine one_div_lt_one_div_of_lt _ _, exact nat.cast_add_one_pos _, simpa }
+
 end nat
```

Comment:
In this discussion on Zulip, Chris Hughes proposed favoring `n ≠ 0` as hypotheses: 

https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/.600.20.3C.20n.60.20or.20.60n.20.E2.89.A0.200.60.3F

In that light, could you rewrite these lemmas so that the avoid the `+1`'s and add the hypotheses `n ≠ 0` and `m ≠ 0`?